### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by cepera2412** [bramkragten/weather-card#130]
Custom element doesn't exist: weather-card.
type: custom:weather-card
entity: weather.openweathermap

gives an error to any parameters.
installed the latest version. master as others have written. in the configuration.yaml wrote "sun". result is one.
does it work at all?
Help!

**Comment by dckiller51** [comment]
hello add manualy to resources 
/hacsfiles/weather-card/weather-card.js

**Comment by johnjoemorgan** [comment]
Thanks

**Comment by johnjoemorgan** [comment]
That worked for a day and now back with the same error - uninstalled, installed - no go.  Placed in the the resource manually - still the same

**Comment by fabseeeee** [comment]
same probleme here .....

**Comment by llranga** [comment]
I'm facing same issue

**Comment by Jo-Jo1987** [comment]
me 2, only mobile version works

**Comment by Hubszo** [comment]
Same problem since last update.

**Comment by sagitt** [comment]
same here. from 1.5.0 to 1.4.6

**Comment by Reynders1** [comment]
Similar problem here. Installed on two different systems with cards set identical, resources defined identical and configuration.yaml set identical.

**Works fine on HA OS**

Version | core-2022.3.8
-- | --
Installation Type | Home Assistant OS
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.9
Operating System Family | Linux
Operating System Version | 5.10.103-v8
CPU Architecture | aarch64

**but not on** 

Version | core-2022.3.8
-- | --
Installation Type | Home Assistant Supervised
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.9
Operating System Family | Linux
Operating System Version | 5.10.0-11-arm64
CPU Architecture | aarch64

**Comment by Imoriarty** [comment]
Tried working some of this out, and got my card to finally show up!

For me the /hacsfiles directory route seems to point from: 

/hacsfiles/weather-card/weather-card.js

to:

/local/community/weather-card/weather-card.js

/local is mapped (for me) to config/www

Might be worth verifying that linkage works correctly / that directory structure is what you have. or swapping out the mappings in the dashboard resources and seeing if any of these other paths work, and troubleshooting from there.

**Comment by Tupsi** [comment]
correcting, I just "redownloaded" with HACS the master branch and that one worked (was using 1.4.6 earlier I think because that was the first showing up during 1st install).
So all good on my end.

**Comment by Jo-Jo1987** [comment]
Issue still persists in supervised version of home assistant

**Comment by Tupsi** [comment]
> 

I disagree. At least I am using that version and it works fine here. Just try reinstalling from MASTER and refresh the browser, should work. I used the version from HACS and removed the extra line one of the posters above said to add by hand, because I ended up with two weather cards.

**Comment by Jo-Jo1987** [comment]
I uninstalled the whole card. And installed the master version.
Cleared cookies. And it really still persists,

**Comment by Jo-Jo1987** [comment]
After a long time somehow the cookies renewed the installation.

Now it works

**Comment by llranga** [comment]
It started working after i upgraded home assistant to version 2022.5.2
